category/serializers.py

```
@@ -1,5 +1,6 @@
 from product.models import Category, Product
 from rest_framework import serializers
+from product.models import Category
```

Ruff (0.12.2)

Redefinition of unused `Category` from line 1

Remove definition: `Category`

(F811)